### ISSUE TITLE
extend CI tests for windows and mac-os platform

### ISSUE THREAD

**Issue by tushar5526** [p5py/p5#433]
**Is your feature request related to a problem? Please describe.**
We need test coverage for multiple platforms as well

**Describe the solution you'd like**
Extending GitHub Actions to support mac-os and Windows

**Comment by AdityyaX** [comment]
hey @tushar5526 ,
Is this issue still open to extend CI test for  windows platform?

**Comment by tushar5526** [comment]
Yes @AdityyaX - this is still up for grabs. You will have to probably research whether there is any way to setup virtual buffers on a windows machine or else some sanity GHAs that attempt p5py installation on windows + python version combinations would suffice.
